frontend/app/components/contextual-alert.tsx

```diff
@@ -0,0 +1,25 @@
+import type { ReactNode } from 'react';
+
+export interface ContextualAlertProps {
+  message_heading: string;
+  children: ReactNode;
+  type: 'warning' | 'success' | 'danger' | 'info';
+}
+
+export function ContextualAlert(props: ContextualAlertProps) {
+  const { message_heading, children, type } = props;
+
+  const alert_color = type === 'warning' ? 'border-l-amber-700' : type === 'danger' ? 'border-l-red-700' : type === 'info' ? 'border-l-cyan-700' : 'border-l-green-700';
+
+  return (
+    <div id="contextual-alert-id" className={`relative min-w-[290px] pl-[16px] sm:pl-[24px]`}>
+      <div className="absolute left-1.5 top-3 bg-white py-1 sm:left-3.5">
+        <img id="image-id" src={`/assets/${type}_img.svg`} alt="" />
```

Comment:
We could alternatively use FontAwesome icons (like we do elsewhere in the application) instead of using images.

If we follow [WET's alerts](https://wet-boew.github.io/wet-boew-styleguide/design/alerts-en.html), we could use the following from `@fortawesome/free-solid-svg-icons`:
- `faCheckCircle` for success alert
- `faCircleInfo` for info alert
- `faExclamationTriangle` for warning alert
- `faExclamationCircle` for danger alert